deploy/demo/catalog-admin-frontend/catalog-admin-frontend-env.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: catalog-admin-frontend
+  labels:
+    app: catalog-admin-frontend
+spec:
+  template:
+    spec:
+      containers:
+        - name: catalog-admin-frontend
+          env:
+            - name: NEXTAUTH_URL
+              value: https://catalog-admin.fellesdatakatalog.digdir.no
```

Comment:
catalog-admin.demo.fellesdatakatalog

Reply:
Fikset ✅ Antok at det gjelder for concept-catalog også?